modules/nodes-nodes-working-master-schedulable.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-working.adoc
+
+[id="nodes-nodes-working-master-schedulable_{context}"]
+= Configuring master nodes as schedulable
+
+You can configure master nodes to be schedulable, meaning that new Pods are allowed for placement on the master nodes. By default, master nodes are not schedulable. However, if your cluster does not contain any worker nodes, then master nodes are marked schedulable by default.
```

Comment:
I think, we should be explicit that from 4.2, we support doing so

Reply:
Thanks, I'll add this.